### ISSUE TITLE
[QUESTION] Dashboard

### ISSUE THREAD

**Issue by gchauvet** [gobo-eiffel/gobo#11]
The TODO list published on gobosoft.com is outdated since a long while.
I'm wondering if would be possible to create tickets of future developments and improvements, by using github labels ?

**Comment by ebezault** [comment]
I'm the one to blame for not keeping the TODO list uptodate.
I'm not familiar with github labels. What are they and how can they be used?

**Comment by gchauvet** [comment]
Take a look here:
https://help.github.com/articles/creating-and-editing-labels-for-issues-and-pull-requests/

**Comment by ebezault** [comment]
So, if I understand correctly, you suggest to use Issues for everything, and attached labels to them as a way to sort them by categories.
Why not.

**Comment by gchauvet** [comment]
Yes :+1:  this trick is used by many projects on GitHub to avoid to maintaining multiple external project tools.

**Comment by jocelyn** [comment]
Side note, using github issue tracker is indeed a way to manage, issues,
tasks, and so on.
And it could also be used with 3rd party services like https://waffle.io/ ,
https://huboard.com and even trello board with zapier integration.
It can help having a board like UI.

-- Jocelyn



On Wed, Sep 2, 2015 at 12:40 PM, Chauvet Guillaume <notifications@github.com

**Comment by gchauvet** [comment]
I've created a dashboard on Trello to manage Gobo project.
